src/ucp/core/ucp_context.c

```diff
@@ -67,6 +68,24 @@ const char *ucp_operation_names[] = {
     [UCP_OP_ID_LAST]          = NULL
 };
 
+static size_t ucp_rndv_frag_sizes[] = {
+    [UCS_MEMORY_TYPE_HOST]         = 512 * UCS_KBYTE,
+    [UCS_MEMORY_TYPE_CUDA]         = 4 * UCS_MBYTE,
+    [UCS_MEMORY_TYPE_CUDA_MANAGED] = 4 * UCS_MBYTE,
+    [UCS_MEMORY_TYPE_ROCM]         = 4 * UCS_MBYTE,
+    [UCS_MEMORY_TYPE_ROCM_MANAGED] = 4 * UCS_MBYTE,
+    [UCS_MEMORY_TYPE_LAST]         = 0
+};
+
+static size_t ucp_rndv_frag_num_elems[] = {
```

Comment:
shy need these global arrays? i'd expect the sizes to be specified by UCP context configuration

Reply:
For default values when user doesn't explicitly specify frag_size for one of the fragment types.